internal/pkg/cli/app_init.go

```diff
@@ -232,6 +234,14 @@ func (o *initAppOpts) validateAppName(name string) error {
 	if err != nil {
 		var noSuchAppErr *config.ErrNoSuchApplication
 		if errors.As(err, &noSuchAppErr) {
+			roleName := fmt.Sprintf("%s-adminrole", name)
+			tags, err := o.iamRoleManager.ListRoleTags(roleName)
+			if err != nil {
+				return nil
+			}
+			if _, hasTag := tags[deploy.AppTagKey]; hasTag {
+				return fmt.Errorf("application named %s already exists in another region", name)
```

Comment:
If we have region of the another application handy by any chance then can we also show that to the customer?

Reply:
As per conversation with @paragbhingre and @Lou1415926. marking this as resolved